sensor_msgs/PointCloud2 cloud
std_msgs/String frame_id_original
std_msgs/String frame_id_desired
float32 x1
float32 x2
float32 y1
float32 y2
float32 z1
float32 z2
---
int64 nr_points_in_area
float32 x_mean
float32 y_mean
float32 z_mean
